end-to-end-test/local/specs/core/resultspage.spec.js

```diff
@@ -0,0 +1,49 @@
+var assert = require('assert');
+var goToUrlAndSetLocalStorage = require('../../../shared/specUtils').goToUrlAndSetLocalStorage;
+var postDataToUrl = require('../../../shared/specUtils').postDataToUrl;
+var parse = require('url-parse');
+var _ = require('lodash');
+
+var useExternalFrontend = require('../../../shared/specUtils').useExternalFrontend;
+
+const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, "");
+
+describe('study select page', function() {
+
+    if (useExternalFrontend) {
+
+        describe('error messaging for invalid study id(s)', function(){
```

Comment:
I am lost here. This file `resutspage.spec.js` has describe `study select page` which belongs to Query Page. So, what does this test actually test? Is there also a test that checks for correct working of the POST method? It is not apparent from the test titles. It seems to only deal with checking of error messages.

Reply:
achhh.  terrible, i forgot to update these descriptions.  they are totally wrong.

Reply:
i will make PR to correct these.